generate-db.js

```diff
@@ -1,78 +1,63 @@
 #!/usr/bin/env node
-const fs = require("node:fs");
-const https = require("node:https");
-const path = require("node:path");
-const zlib = require("node:zlib");
+import { readFile, writeFile } from 'node:fs/promises';
+import { gunzip } from 'node:zlib';
+import { join } from 'node:path';
+import { promisify } from 'node:util';
+
+const gunzipAsync = promisify(gunzip);
 
-// TODO(@thunder-coding): Do not hardcode list of known architectures.
```

Comment:
Thanks for reducing one TODO off my TODO list, really appreciate it!

Reply:
Done!